site/content/installation/ngf-images/using-the-jwt-token.md

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
This file is a duplicate of your other JWT one.

Reply:
deleted